user-interface/7004-POSI-positions.md

```diff
@@ -1 +1,57 @@
-# View my Position(s)
\ No newline at end of file
+# View my Position(s)
+When a party first trades on a market they have a "position". When their exposure returns to 0 (as the result or orders or settlement) the position has gone from being **open** to **closed**.
+
+On some exchanges having an active order on a market (but no-open volume) is also defined as an open position. On Vega these will not come back in the positions API (unless you already traded this market, where it will come back as a position with no open volume).
+
+On some exchanges, one can have multiple "positions" on a single market, one after another, each starts and ends when open volume is zero or the direction of your exposure changes. On Vega a party only has one "position" on a market regardless of how many times they have netted of. Therefore on Vega it is also impossible to "force open" a position, being both long and short at the same time (although a user could use different keys).
+
+Each position has both a realised and un-realised [profit and loss](9001-DATA-data_display.md#profit-and-loss). The unrealised PnL is an indication of the profit and lost might be realised if the position's open volume were reduced to zero. Realised PnL is a sum of all the realised PnL to date.
+
+When a position is open it has a volume weighted average "open" price, the size + direction of your exposure (long or short), an indication the positions margin requirements (more on this later) 
+ 
+ Note: as with orders users should be able to customise what data appears on the positions view. As well as create your own filters, groups and sorts on the position data. The following assumes two positions view, one for open/active/live positions, and one for all positions so that users can see realised PnL, and have somewhere to go if they were expecting to see an open position.
```

Comment:
```suggestion
When a position is open it has a volume weighted average "open" price, the size + direction of your exposure (long or short), an indication the positions margin requirements (more on this later).
 
Note: as with orders, users should be able to customise what data appears on the positions view. As well as creating their own filters, groups and sorts on the position data. The following assumes two positions views, one for open/active/live positions, and one for all positions so that users can see realised PnL, and have somewhere to go if they were expecting to see an open position.
```